src/utils/options.ts

```diff
@@ -9,13 +9,11 @@ import { toArray } from "@/utils/array";
  * @returns Comma separated values as string.
  */
 
-export const statusColor = (options: Option[], values?: string | string[]): string => {
-  if (!values) return "";
-  if (options.length === 0) return toArray(values).join(",");
+export const getMetaData = (options: Option[], values?: string | string[]): any => {
```

Comment:
This change isn't required, but I’d like to clarify if getMetaData is intended to return multiple types. If so, using any is acceptable, but i have the next question:
Could we define a more specific return type instead? For example, if the function is expected to return a specific structure or type, let's specify that.

Also, if the function may return undefined, we could use a union type. Here's a quick suggestion:
`export const getMetaData = (options: Option[], values?: string | string[]): YourExpectedReturnType | undefined => {
  if (!values) return undefined;
  if (options.length === 0) return undefined;
  // your logic here
};
`
As I mentioned it's just in case we know what is the expected return.

Reply:
The 'any' type comes from a type definition made a long time ago:
export type Option = {
  title: string;
  value: OptionValue;
  meta?: any;
};

It seems like 'meta' was not implemented in the dropdown initially. In this case, I'm using it to pass extra data to render, specifically flags. 
In other scenarios, which we don't have right now, 'meta' could represent a status, an extra string like a description, or something else. 
Therefore, I think it's reasonable to keep 'meta' as 'any'. As a result, the default value should be 'undefined'.
![image](https://github.com/user-attachments/assets/eda0aab1-68d3-499c-81b1-1bb972eb88d6)